src/pages/AddItem.js

```diff
@@ -88,6 +144,8 @@ const AddItem = ({ firestore }) => {
           </label>
         </div>
       </form>
+
+      {duplicate ? <ItemError /> : null}
```

Comment:
Duplicate error message worked like a charm when I had it to show when there was nothing added to the input. However when I tried connecting to the data base is when it started breaking.

Also decided to create it as it's own component - maybe we can refactor it to use it for other error messages